crypto/aes/asm/bsaes-armv7.pl

```diff
@@ -1119,10 +1119,10 @@ sub bitslice_key {
 .extern AES_cbc_encrypt
 .extern AES_decrypt
 
-.global	bsaes_cbc_encrypt
-.type	bsaes_cbc_encrypt,%function
+.global	ossl_bsaes_cbc_encrypt
```

Comment:
I was a bit scared to do this - how do we know if we are breaking some of these?

Reply:
We don't until someone tries to do an ARMv7 build.
A cross compile for an ARMv7 target would be indicative of some problems.

Reply:
Not sure I see the problem...  this is an internal symbol, so as long as we refer to it by that same name from everywhere else in the code, this should be fine.

The name itself isn't ARMv7 only, it's also there in the x86_64 assembler.  It should therefore be quite easy to build on anything x86_64, and do `nm libcrypto.a` and grep for `bsaes_cbc_encrypt` to see if there's any un-prefixed reference left.